app/views/activities/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :title, 'All Activity' %>
+
+<% content_for :breadcrumbs do %>
+  <nav>
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item">
+        <%= link_to 'Dashboard', project_path(current_project) %>
+      </li>
+      <li class="breadcrumb-item active">All Activity</li>
+    </ol>
+  </nav>
+<% end %>
+
+<div class="content-container mt-3 <%= 'pt-3' if @activities_groups.any? %>">
+  <% if @activities_groups.any? %>
+    <div class="container">
+      <div class="activities-groups-container" data-behavior="activities-groups-container" data-page="<%= params[:page].present? ? params[:page] : 1 %>" data-url="<%= project_activities_path(current_project, format: :js) %>">
```

Comment:
Let's evaluate the `data-page` at the controller level, instead of in this view. Something like:
```
@page = params[:page].present? ? params[:page] : 1
```